### PR TITLE
Text filtering

### PR DESCRIPTION
**Pull request by FlattestGuitar**
:cl: Flatty, Ralta and FeiH
add: REEing or Yeeting now gives you a witty line instead of letting you spout that garbage.
/:cl:

I tested this on this file: https://raw.githubusercontent.com/dwyl/english-words/master/words.txt which is over 466 thousand words from the English language. It only triggered on one: `Ree`. Not sure what the word means in English, but I'm afraid to check.

I also added a yeet regex now. It will block a `yet` with more than one `e`, so watch yourself.

It doesn't deal brain damage anymore. Just stops you from saying the thing.

I need help writing more witty lines.

### CODE REVIEW

**Comment by AffectedArc07** [comment]
can we get this with tildes too?

File: code/_globalvars/lists/names.dm [hunk 1]
```diff
@@ -21,5 +21,6 @@ GLOBAL_LIST_INIT(posibrain_names, world.file2list("strings/names/posibrain.txt")
 GLOBAL_LIST_INIT(verbs, world.file2list("strings/names/verbs.txt"))
 GLOBAL_LIST_INIT(adjectives, world.file2list("strings/names/adjectives.txt"))
 GLOBAL_LIST_INIT(dream_strings, world.file2list("strings/dreamstrings.txt"))
+GLOBAL_LIST_INIT(retard_lines, world.file2list("strings/retard_lines.txt"))
```
Comment (Ziiro) [inline code/_globalvars/lists/names.dm]: `retard_lines`

how problematic uwu

**Comment by Ralta** [comment]
`You re-evaluate your life choices.`
`You try hard to remember how many times you were dropped as a child.`
`You hope that this will win you the Darwin award.`
`Working so hard to formulate a sentence has left your brain tired.`
`You feel like a screw is loose...but you don't have any screws.`
`You feel as if you're a few sandwiches short of a picnic.`
`You feel as though you're a few cards short of a deck.`
`You feel like you're a few beers short of a six-pack.`
`Your current IQ feels like room temperature.`
`You feel dumber than a box of Donk pockets.`

**Comment by Ralta** [comment]
I'll try and come up with some more later.

**Comment by spoonitate** [comment]
`Your words are as empty as your head.`
`It feels like rats made a home in your skull.`
`Your vocal cords feel like they shredded themselves the moment you tried to make a sound.`
`Your butt must feel jealous of your mouth from all the shit that comes out of it.`
`You punch yourself. You don't know why, but you feel like it was justified.`
`Your head aches as images of screaming frogs suddenly fill your vision.`
`All you can manage is a strangled croak before you succumb to a violent coughing fit.`

File: code/modules/mob/living/say.dm [hunk 2]
```diff
@@ -140,6 +140,12 @@ GLOBAL_LIST_INIT(department_radio_keys, list(
 	if(!can_speak_vocal(message))
 		to_chat(src, "<span class='warning'>You find yourself unable to speak!</span>")
 		return
+		
+	if(check_retard_speech(message))
+		var/reeee = pick(GLOB.retard_lines)
+		to_chat(src, "<span class='warning'>[reeee]</span>")
+		adjustBrainLoss(20, 150)//20 damage, max 150, so 75%
```
Comment (SamHPurp) [inline code/modules/mob/living/say.dm]: No. This is going to just be abused be bored dickheads. Absolutely not.

**Comment by SamHPurp** [comment]
Can you profile this, I don't want a huge amount of drain on the CPU for something largely useless.

If it doesn't appear heavy, then lets merge it.

**Pull request merged**